piet-wgsl/shader/coarse.wgsl

```diff
@@ -235,8 +234,7 @@ fn main(
             let x1 = clamp(i32(path.bbox.z) - i32(bin_tile_x), 0, i32(N_TILE_X));
             let y1 = clamp(i32(path.bbox.w) - i32(bin_tile_y), 0, i32(N_TILE_Y));
             sh_tile_width[local_id.x] = u32(x1 - x0);
-            sh_tile_x0[local_id.x] = u32(x0);
-            sh_tile_y0[local_id.x] = u32(y0);
+            sh_tile_x0y0[local_id.x] = u32(x0) | u32(y0 << 16u);
```

Comment:
Should this `16` be a constant defined somewhere?

It's probably fine as-is.

Reply:
Seems like a common enough pattern to not require a definition.